go/stellar/stellarsvc/service_test.go

```diff
@@ -763,7 +763,7 @@ func TestRequestPaymentOutsideCurrency(t *testing.T) {
 	require.Nil(t, details.Asset)
 	require.NotNil(t, details.Currency)
 	require.Equal(t, stellar1.OutsideCurrencyCode("USD"), *details.Currency)
-	require.Equal(t, "$8.20 USD", details.AmountDescription)
+	require.Equal(t, "$8.2 USD", details.AmountDescription)
```

Comment:
This is what we want?  $8.2 USD?

Reply:
I just talked to coyne, I misunderstood. The ticket should be dropped in favor of always doing two decimal points for USD like we were doing already.

Reply:
yeah, sorry if I wasn't clear.

 I was trying to say in my long post above there are 3 _imaginable_ ways we'd want to format something. In the 2-digit case it should always be 2-digits; I'm agreeing with you two about fiats always getting decimals and I just relayed that to Cécile and Adam.

In the smart digit case it shouldn't use any unnecessary zeroes.

And in the full digit case (which I'm not sure we'll ever use for something like XLM) it would give full precision always.